pygpudrive/datatypes/metadata.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from dataclasses import dataclass
+import gpudrive
+
+
+class Metadata:
+    """A class to represent object metadata (eg: sdc_flag)
+    Initialized from metadata_tensor (src/bindings). For details, see
+    `MetaData` in src/types.hpp
```

Comment:
Can you add the shapes of the masks in the docstring? (think these will be NumWorlds, MaxAgentCount)